src/jdk.localedata/share/classes/sun/text/resources/ext/FormatData_ja.java

```diff
@@ -87,7 +87,7 @@ protected final Object[][] getContents() {
         return new Object[][] {
             { "japanese.FirstYear",
                 new String[] {  // first year name
-                    "\u5143",   // "Gan"-nen
+                    "元",   // "Gan"-nen
```

Comment:
```suggestion
                    "元",       // "Gan"-nen
```
Preserve comment alignment?